src/main/resources/data.harness.properties

```diff
@@ -12,36 +12,36 @@ find.groups.results.empty.description={"groupResults":[{"idIndex":"843904","type
 find.groups.results.failure={ "groupResults": [], "resultMetadata": { "params": null, "resultCode": "SUCCESS", "resultCode2": null, "success": "T", "resultMessage": "" }, "responseMetadata": { "resultWarnings": "", "millis": "7", "serverVersion": "2.2.2" } }
 
-ws.subject.success.uid={"resultCode":"SUCCESS","success":"T","identifierLookup":"testiwta","id":"99997010","name":"Testf-iwt-a TestIAM-staff","sourceId":"UH core LDAP","attributeValues":["testiwta","Testf-iwt-a TestIAM-staff","TestIAM-staff","Testf-iwt-a","","eduPersonOrgDN=uhsystem,eduPersonAffiliation=staff","staff","uhsystem",""]}
-ws.subject.success.uhuuid={"resultCode":"SUCCESS","success":"T","identifierLookup":null,"id":"99997010","name":"Testf-iwt-a TestIAM-staff","sourceId":"UH core LDAP","attributeValues":["testiwta","Testf-iwt-a TestIAM-staff","TestIAM-staff","Testf-iwt-a","","eduPersonOrgDN=uhsystem,eduPersonAffiliation=staff","staff","uhsystem",""]}
+ws.subject.success.uid={"resultCode":"SUCCESS","success":"T","identifierLookup":"testiwta","id":"99997010","name":"Testf-iwt-a TestIAM-staff","sourceId":"UH core LDAP","attributeValues":["testiwta","Testf-iwt-a TestIAM-staff","TestIAM-staff","Testf-iwt-a","","eduSubjectOrgDN=uhsystem,eduSubjectAffiliation=staff","staff","uhsystem",""]}
+ws.subject.success.uhuuid={"resultCode":"SUCCESS","success":"T","identifierLookup":null,"id":"99997010","name":"Testf-iwt-a TestIAM-staff","sourceId":"UH core LDAP","attributeValues":["testiwta","Testf-iwt-a TestIAM-staff","TestIAM-staff","Testf-iwt-a","","eduSubjectOrgDN=uhsystem,eduSubjectAffiliation=staff","staff","uhsystem",""]}
 ws.subject.subject.uid.not.found={"resultCode":"SUBJECT_NOT_FOUND","success":"F","identifierLookup":"invalid-uid","id":null,"name":null,"sourceId":null,"attributeValues":null}
 ws.subject.subject.uhuuid.not.found={"resultCode":"SUBJECT_NOT_FOUND","success":"F","identifierLookup":null,"id":"11111111","name":null,"sourceId":null,"attributeValues":null}
-ws.subject.success.null.values={ "resultCode": null, "success": "T", "identifierLookup": "testiwta", "id": "99997010", "name": "Testf-iwt-a TestIAM-staff", "sourceId": "UH core LDAP", "attributeValues": [ "testiwta", "Testf-iwt-a TestIAM-staff", null, null, "", "eduPersonOrgDN=uhsystem,eduPersonAffiliation=staff", "staff", "uhsystem", "" ] }
+ws.subject.success.null.values={ "resultCode": null, "success": "T", "identifierLookup": "testiwta", "id": "99997010", "name": "Testf-iwt-a TestIAM-staff", "sourceId": "UH core LDAP", "attributeValues": [ "testiwta", "Testf-iwt-a TestIAM-staff", null, null, "", "eduSubjectOrgDN=uhsystem,eduSubjectAffiliation=staff", "staff", "uhsystem", "" ] }
```

Comment:
These lines will have to stay as `eduPersonOrgDN` and `eduPersonAffiliation` because they come directly from Grouper and cannot be changed.